src/variables.js

```diff
@@ -152,10 +151,9 @@ module.exports = {
 				if (JSON.stringify(oldValue) !== JSON.stringify(newValue)) {
 					this.store[key] = newValue
 					if (key === 'clips') {
+						this.initVariables()
 						this.updateVariable('clip_name', this.store.clips[this.store.clipID].name)
-						for (var i = 0; i < this.config.maxClips; i++) {
-							this.updateVariable('clip_' + (i+1), this.store.clips[i]?.name)
-						}
+						this.store.clips.forEach((clip, index) => this.updateVariable(`clip_${index + 1}_name`, clip.name))
```

Comment:
update each variable with the name. Template strings are used as well according to the previous comment.